src/_nebari/upgrade.py

```diff
@@ -705,6 +705,20 @@ def _version_specific_upgrade(
         return config
 
 
+class Upgrade_2024_3_1(UpgradeStep):
+    version = "2024.3.1"
+
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        rich.print("\n ⚠️  Warning ⚠️")
+        rich.print(
+            "-> Please run [green]mv stages stages.old[/green] so that we can keep a backup of the original Terraform scripts and regenerate an updated set of them for your deployment."
```

Comment:
Take this comment or leave it, but I don't feel like even this is necessary for this release. I feel like this could be as simple as printing "upgrading to 2024.2.1" or something like that. I am still approving in case you decide to keep this, but I don't think we need it.

Reply:
It's probably not because we're not deprecating anything. I took your suggestion and incorporated it.